unic/ucd/hangul/tests/basic_tests.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+extern crate unic_ucd_hangul;
+use unic_ucd_hangul::hangul;
+
+#[test]
+fn test_is_syllable() {
+    assert!(hangul::is_syllable('갏'));
+    assert!(!hangul::is_syllable('A'));
+    assert!(!hangul::is_syllable('韩'));
+}
+
+#[test]
+fn test_decompose() {
+    let mut decomposed_lv = vec![];
+    {
+        let mut collect_decomposed = |chr| {
+            decomposed_lv.push(chr);
+        };
+        hangul::decompose('쮀', &mut collect_decomposed);
+    }
+    assert_eq!(decomposed_lv, ['ᄍ', 'ᅰ']);
+
+    let mut decomposed_lvt = vec![];
+    {
+        let mut collect_decomposed = |chr| {
+            decomposed_lvt.push(chr);
+        };
+        hangul::decompose('퓛', &mut collect_decomposed);
+    }
+    assert_eq!(decomposed_lvt, ['ᄑ', 'ᅱ', 'ᆶ']);
+}
+
+#[test]
+fn test_compose() {
```

Comment:
I missed this test file before. Looks good. Maybe we can add more function to it that decomposes and the composes, and use that in the docblock tests?

(Or you can leave the test additions fro another PR. Up to you.)

Reply:
Added a new test case `test_decompose_then_compose` for this.
Also used the new test case in the docblock tests.